packages/common/src/types/ccdAmount.ts

```diff
@@ -6,7 +10,30 @@
 export class CcdAmount {
     microCcdAmount: bigint;
 
-    constructor(microCcdAmount: bigint) {
+    /**
+     * Constructs a CcdAmount and checks that it is valid, can accept a number, string, big or bigint.
+     * Accepts strings with decimal seperators as both commas and decimals.
+     *
+     * @param microCcdAmount Theamount of micro CCD as a number, string, big or bigint.
+     * @throws If the number of commas in the input string exceeds 1
```

Comment:
Does this apply also to the `dot` as a decimal separator? Then, `@throws If the number of decimal seperators in the input string exceeds 1`